lldb/source/Plugins/Process/Windows/Common/ProcessWindows.cpp

```diff
@@ -406,7 +406,7 @@ void ProcessWindows::RefreshStateAfterStop() {
                m_session_data->m_debugger->GetProcess().GetProcessId(), pc, id);
 
       stop_info = StopInfo::CreateStopReasonWithWatchpointID(
-          *stop_thread, id, m_watchpoints[id].address);
+          *stop_thread, id);
```

Comment:
Ah yikes, I believe I broke this in https://reviews.llvm.org/D147816 and that `addr_t` is being treated like a boolean now :(